alpha/zitadel/values.yaml

```diff
@@ -0,0 +1,173 @@
+# Default values for zitadel.
+zitadel:
+  # The ZITADEL config under configmapConfig is written to a Kubernetes ConfigMap
+  # See all defaults here:
+  # https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml
+  configmapConfig:
+    Database:
+      cockroach:
+        Host: "crdb-public"
+        User:
+          SSL:
+            Mode: "verify-full"
+        Admin:
+          SSL:
+            Mode: "verify-full"
+
+    Machine:
+      Identification:
+        Hostname:
+          Enabled: true
+        Webhook:
+          Enabled: false
+
+    # this is for Argo CD OIDC
+    # ref: https://github.com/zitadel/zitadel-charts/discussions/116
+    # ref: https://github.com/argoproj/argo-cd/discussions/11855#discussioncomment-5586777
+    Actions:
+      groupsClaim:
+        allowedToFail: true
+        timeout: 10
+        script: |
+          function groupsClaim(ctx, api) {
+            if (ctx.v1.user.grants === undefined || ctx.v1.user.grants.count == 0) {
+              return;
+            }
+
+            let grants = [];
+            ctx.v1.user.grants.grants.forEach(claim => {
+              claim.roles.forEach(role => {
+                  grants.push(role)  
+              })
+            })
+
+            api.v1.claims.setClaim('groups', grants)
+          }
+
+    ## If you want to setup ZITADEL with a service account
+    ## instead of a human admin user, comment the following
+    ## in by deleting each lines first hash and space
+    # FirstInstance:
+    #   # path used for volume mounts and to write the secret
+    #   MachineKeyPath: /machinekey/zitadel-admin-sa.json
+    #   Org:
+    #     Machine:
+    #       Machine:
+    #         # Creates a service account with the name zitadel-admin-sa,
+    #         # which results in a secret 'zitadel-admin-sa' with a key 'zitadel-admin-sa.json'
+    #         Username: zitadel-admin-sa
+    #         Name: Admin
+    #       MachineKey:
+    #         Type: 1
+
+  # The ZITADEL config under secretConfig is written to a Kubernetes Secret
+  # See all defaults here:
+  # https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml
+  secretConfig:
+
+  # Reference the name of a secret that contains ZITADEL configuration.
+  # The key should be named "config-yaml".
+  configSecretName:
+
+  # ZITADEL uses the masterkey for symmetric encryption.
```

Comment:
Maybe?
  
  # ZITADEL uses symmetric encryption.
  # You can generate the encryption key using: 'tr -dc A-Za-z0-9 </dev/urandom | head -c 32'
  # or reference it via a secret. the name "masterkey" is hardcoded unfortunately. 
  # One of either the key or secret-name must be provided.
  masterkey: ""
  # Name of the secret which contains the key.
  masterkeySecretName: ""